packages/web-components/fast-foundation/src/listbox/listbox.element.ts

```diff
@@ -0,0 +1,114 @@
+import {
+    attr,
+    css,
+    ElementStyles,
+    nullableNumberConverter,
+    observable,
+} from "@microsoft/fast-element";
+import { Listbox } from "./listbox";
+
+/**
+ * A Listbox Custom HTML Element.
+ * Implements the {@link https://w3c.github.io/aria/#listbox | ARIA listbox }.
+ *
+ * @public
+ */
+export class ListboxElement extends Listbox {
+    /**
+     * The attribute to set the size property.
+     *
+     * @remarks
+     * HTML Attribute: `size`.
+     *
+     * The value of the `size` attribute is reflected by the `size` property.
+     *
+     * @public
+     */
+    @attr({
+        attribute: "size",
+        converter: nullableNumberConverter,
+        mode: "fromView",
+    })
+    public sizeAttribute: number;
```

Comment:
`sizeAttribute` feels like an odd name for a prop, not that I necessarily have a better suggestion...

Reply:
This is now updated to remove `sizeAttribute`, in favor of only using the `@attr size` property.